javascript-forms/form-events.js

```diff
@@ -2,58 +2,30 @@ function handleFocus(event) {
   console.log('event.target.name', event.target.name);
 }
-const $focus = document.querySelector('#user-name');
-$focus.addEventListener('focus', handleFocus);
+const $focusName = document.querySelector('#user-name');
+$focusName.addEventListener('focus', handleFocus);
+const $focusEmail = document.querySelector('#user-email');
+$focusEmail.addEventListener('focus', handleFocus);
+const $focusInput = document.querySelector('#user-message');
+$focusInput.addEventListener('focus', handleFocus);
 
 function handleBlur(event) {
   console.log('blur event fired');
   console.log('event.target.name', event.target.name);
 }
-const $blur = document.querySelector('#user-name');
-$blur.addEventListener('blur', handleBlur);
+const $blurName = document.querySelector('#user-name');
+$blurName.addEventListener('blur', handleBlur);
+const $blurEmail = document.querySelector('#user-email');
+$blurEmail.addEventListener('blur', handleBlur);
+const $blurInput = document.querySelector('#user-message');
+$blurInput.addEventListener('blur', handleBlur);
 
 function handleInput(event) {
   console.log('value of name:', event.target.value);
 }
-const $input = document.querySelector('#user-name');
-$input.addEventListener('input', handleInput);
-
-function handleFocusEmail(event) {
-  console.log('focus event fired');
-  console.log('event.target.email', event.target.name);
-}
-const $focusEmail = document.querySelector('#user-email');
-$focusEmail.addEventListener('focus', handleFocusEmail);
-
-function handleBlurEmail(event) {
-  console.log('blur event fired');
-  console.log('event.target.email', event.target.name);
-}
-const $blurEmail = document.querySelector('#user-email');
-$blurEmail.addEventListener('blur', handleBlurEmail);
-
-function handleInputEmail(event) {
-  console.log('value of email:', event.target.value);
-}
+const $inputName = document.querySelector('#user-name');
+$inputName.addEventListener('input', handleInput);
 const $inputEmail = document.querySelector('#user-email');
-$inputEmail.addEventListener('input', handleInputEmail);
-
-function handleFocusMessage(event) {
-  console.log('focus event fired');
-  console.log('event.target.message', event.target.name);
-}
-const $focusMessage = document.querySelector('textarea');
-$focusMessage.addEventListener('focus', handleFocusMessage);
-
-function handleBlurMessage(event) {
-  console.log('blur event fired');
-  console.log('event.target.message', event.target.name);
-}
-const $blurMessage = document.querySelector('textarea');
-$blurMessage.addEventListener('blur', handleBlurMessage);
-
-function handleInputMessage(event) {
-  console.log('value of message:', event.target.value);
-}
-const $inputMessage = document.querySelector('textarea');
-$inputMessage.addEventListener('input', handleInputMessage);
+$inputEmail.addEventListener('input', handleInput);
+const $inputInput = document.querySelector('#user-message');
+$inputInput.addEventListener('input', handleInput);
```

Comment:
You shouldn't have so many `querySelector`s. You should only have 3. You just query for each input once, and use those variable names to attach all 3 events to them